docs/public-networks/how-to/connect/manage-peers.md

```diff
@@ -26,6 +26,15 @@ You can limit peers to reduce the bandwidth, CPU time, and disk access Besu uses
 To reduce the maximum number of peers, use the
 [`--max-peers`](../../reference/cli/options.md#max-peers) option. The default is 25.
 
+!!! caution
+
+    The miminum number of peers is set by the `--xp2p-peer-lower-bound` option, which also has a default of 25.
```

Comment:
Should the "x" be capitalised? `--Xp2p-peer-lower-bound`

Reply:
Updated capitalization and added to the CLI description